app/src/androidTest/java/org/fossasia/susi/ai/signup/SignUpActivityTest.kt

```diff
@@ -63,4 +64,12 @@ class SignUpActivityTest {
         onView(withId(R.id.signUp)).perform(scrollTo())
         onView(withId(R.id.signUp)).check(matches(isDisplayed()))
     }
+
+    @Test
+    fun `testSignupPassword`() {
+        Timber.d("running password test..")
+        val password = "123345602"
+        val result = mActivityRule.activity.passwordTest(password)
+        assertEquals("Pattern did not match", false, result)
+    }
```

Comment:
This is a unit test, not an instrumentation test

Reply:
Okk, I will update it